src/arc_utilities/tf2wrapper.py

```diff
@@ -0,0 +1,138 @@
+#! /usr/bin/env python
+
+import geometry_msgs.msg
+import rospy
+import tf2_ros
+from arc_utilities import transformation_helper
+
+
+class TF2Wrapper:
```

Comment:
:+1: 

I always find tf2_ros pretty verbose. I'll try this out!

Reply:
yea I love this class